app/src/main/res/values/ids.xml

```diff
@@ -1358,4 +1358,9 @@
   <item type="id" name="controller_emojiLayoutEmojiTrending" />
   <item type="id" name="controller_emojiLayoutReactions" />
   <item type="id" name="controller_reactionsPicker" />
+  <item type="id" name="btn_convertToBroadcastGroup" />
+  <item type="id" name="btn_convertBroadcastGroup" />
+  <item type="id" name="convertBroadcastGroup" />
+  <item type="id" name="convertBroadcastGroup2" />
+  <item type="id" name="convertBroadcastGroup3" />
```

Comment:
Better to include `btn_` in prefix for consistency and suffix for use case clarity.